src/components/Other/Home.js

```diff
@@ -19,28 +19,45 @@ const Home = () => {
 
   return (
     <Grid container justify="center">
-      <Grid item xs={10}>
-        <Typography gutterBottom variant="h3" component="h1">
+      <Grid item xs={10} align="center">
+        <Typography
+          gutterBottom
+          variant="h5"
+          component="h1"
+          style={{margin: '20px', fontSize: '25px'}}
```

Comment:
These inline `style=...` properties should be pulled out into a `styles` object for the whole component